### PR TITLE
add qfieldsync layer properties doc

### PR DESCRIPTION
**Pull request by lucienicolier**

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-2547 Document maximum items in relation editor widget](https://app.clickup.com/t/2192114/QF-2547)

File: documentation/get-started/tutorials/get-started-qfs.en.md [hunk 1]
```diff
@@ -76,6 +76,16 @@ Remove
     layer is used in the basemap and will not be available in the
     packaged project.
 
+Properties
+
+:   There are some set up possibilities in the properties.
+
+1. Lock geometries
+2. Set up the attachment default naming, please refer to the [Configurable picture path](../picture/#configurable-picture-path)
+3. Set up the number of maximum of items shown in the relation editor widget
```
Comment (m-kuhn) [inline documentation/get-started/tutorials/get-started-qfs.en.md:85]:
```suggestion
3. Set up the maximum number of items shown in the relation editor widget
```

File: documentation/get-started/tutorials/get-started-qfs.en.md [hunk 2]
```diff
@@ -76,6 +76,16 @@ Remove
     layer is used in the basemap and will not be available in the
     packaged project.
 
+Properties
+
+:   There are some set up possibilities in the properties.
```
[the remark this answers is not in the document]
Comment (suricactus) [reply documentation/get-started/tutorials/get-started-qfs.en.md:81]:
```suggestion
:   There are some additional options to fine tune your QField project in the properties.
```

File: documentation/get-started/tutorials/get-started-qfs.en.md [hunk 3]
```diff
@@ -76,6 +76,16 @@ Remove
...
+
+1. Lock geometries
+2. Set up the attachment default naming, please refer to the [Configurable picture path](../picture/#configurable-picture-path)
```
[the remark this answers is not in the document]
Comment (m-kuhn) [reply documentation/get-started/tutorials/get-started-qfs.en.md:84]: can you adjust it here please, @lucienicolier ?

[the remark this answers is not in the document]
Comment (lucienicolier) [reply documentation/get-started/tutorials/get-started-qfs.en.md:84]: no need, with @nirvn we decided to keep pictures, for now, not to break any path

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**